e2e/Cargo.lock

```diff
@@ -1,3 +1,5 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
```

Comment:
These lines are present in this file already, please either remove these or remove line 20-21

Reply:
OK. I'll regenerate the lock file and it should converge.